app/gui/view/component-browser/src/lib.rs

```diff
@@ -71,7 +71,7 @@ impl component::Model for Model {
         let scene = &app.display.default_scene;
         shapes_order_dependencies! {
             scene => {
-                component_list_panel::background -> documentation::overlay;
+                component_list_panel::background -> display::shape::compound::rectangle::shape;
```

Comment:
```suggestion
                component_list_panel::background -> display::shape::compound::rectangle;
```

I think `::shape` is equivalent but not needed.